modules/tinymce/src/themes/silver/main/ts/ui/dropdown/CommonDropdown.ts

```diff
@@ -119,18 +167,32 @@ const renderCommonDropdown = <T>(
       matchWidth: true,
       useMinWidth: true,
 
-      // TODO: Not quite working. Can still get the button focused.
+      // When the dropdown opens, if we are in search mode, then we want to
+      // focus our searcher.
+      onOpen: (anchor, dropdownComp, tmenuComp) => {
+        if (spec.searchable) {
+          focusSearchField(tmenuComp);
+        }
+      },
+
       dropdownBehaviours: Behaviour.derive([
         ...spec.dropdownBehaviours,
         DisablingConfigs.button(() => spec.disabled || sharedBackstage.providers.isDisabled()),
         ReadOnly.receivingConfig(),
+        // INVESTIGATE: There was a old comment here about something not quite working, and that
```

Comment:
Does it need a ticket? Or any other investigative comments in this PR.

Reply:
I can make one. This was a very old comment. I just put a note next to it that we don't really have much context about what it was about.

- [ ] make a ticket for this investigation.

Reply:
Updated: links to TINY-9012